packages/viz/src/lib/style/helpers/size-bins-style.ts

```diff
@@ -177,7 +178,17 @@ function calculateWithBreaks(
   };
 }
 
-function validateParameters(options: SizeBinsOptionsStyle) {
+function validateParameters(
+  options: SizeBinsOptionsStyle,
+  geometryType: GeometryType
+) {
+  if (geometryType === 'Polygon') {
+    throw new CartoStylingError(
+      "Polygon layer doesn't support sizeCategoriesStyle",
+      stylingErrorTypes.GEOMETRY_TYPE_UNSUPPORTED
+    );
+  }
+
   if (options.bins < 1) {
```

Comment:
I think we should also limit the upper bins option. We could potentially have an infinite number of bins, not as in colors (limited by palettes), but I think we should be homogeneous on this. 

So I propose limiting the upper to 7 too. In any case, having more differences is probably non-sense and really difficult for the eye to perceive...